filesystems/drs/src/main/scala/cromwell/filesystems/drs/DrsPathBuilder.scala

```diff
@@ -15,9 +15,24 @@ case class DrsPathBuilder(fileSystemProvider: DrsCloudNioFileSystemProvider) ext
 
   override def name: String = "DRS"
 
+  /**
+    * Unlike other cloud providers where directories are identified with a trailing slash at the end like `gs://bucket/dir/`,
+    * DRS has a concept of bundles for directories (not supported yet). This sanitizes the path by removing the trailing '/'(s)
+    * so that the CloudNioFileSystemProvider & CloudNioPath does not treat a DRS path ending with '/' as a directory, otherwise
+    * it's size is returned as 0
+    */
+  private def removeTrailingSlashes(path: String): String = {
```

Comment:
This extra sanitization is open for debate. If a drs input has trailing `/`, and this sanitization doesn't take place, the checks in `CloudNioFileSystemProvider` to see if the path is a directory will return true and hence the size of the input file will be 0. But since DRS has a different concept for directories, we can debate that if a path has trailing `/` it can be counted as an error/mistake and hence the size should be returned 0 rather than sanitizing the input.